annotationProcessor/build.gradle

```diff
@@ -1,8 +1,8 @@
 plugins {
-    id "java"
-    id "tanagra.java-conventions"
-    id "io.spring.dependency-management" version "1.0.11.RELEASE"
-    id "org.springframework.boot" version "2.5.1"
+    id 'java'
```

Comment:
Using single quotes in gradle files for strings where no variable substitution is expected